shapes/neurosciencegraph/datashapes/core/workflowengine/schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "@context": [
+    "https://incf.github.io/neuroshapes/contexts/schema.json",
+    {
+      "this": "https://neuroshapes.org/dash/softwareagent/shapes/"
+    }
+  ],
+  "@id": "https://neuroshapes.org/dash/workflowengine",
+  "@type": "nxv:Schema",
+  "imports": [
+    "https://provshapes.org/commons/agent"
+  ],
+  "shapes": [
+    {
+      "@id": "this:WorkflowEngineShape",
+      "@type": "sh:NodeShape",
+      "description": "Software agent which executes automated workflows.",
+      "targetClass": "nsg:WorkflowEngine",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "https://provshapes.org/commons/agent/shapes/AgentShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:name",
+              "name": "Name of the workflow that engine was running.",
+              "datatype": "xsd:string",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "schema:description",
+              "name": "Optional description of the workflow.",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "schema:version",
+              "name": "Version of the workflow engine.",
+              "datatype": "xsd:string",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "prov:actedOnBehalfOf",
+              "name": "Actual user agent who triggered the execution of the workflow.",
+              "class": "prov:Agent",
```

Comment:
This should be the same as `_createdBy`/`_updatedBy` IRI. Not sure if we'll have a full blown `prov:Agent` entity there or I should replace it with `sh:IRI`.